<ui version="4.0" >
<comment>
    Copyright (C) 2004-2010  Martin Jolicoeur (snmpb1@gmail.com) 

    This file is part of the SnmpB project 
    (http://sourceforge.net/projects/snmpb)

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 2 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see http://www.gnu.org/licenses/.
</comment>
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Replace Text</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabel2" >
       <property name="text" >
        <string>R&amp;eplace:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>comboReplace</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>&amp;Find:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>comboFind</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="comboFind" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="comboReplace" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="ButtonGroup1" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkWords" >
        <property name="text" >
         <string>&amp;Whole words only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCase" >
        <property name="text" >
         <string>Case &amp;sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBackward" >
        <property name="text" >
         <string>Search &amp;backward</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="buttonReplace" >
         <property name="text" >
          <string>&amp;Replace</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="buttonFindNext" >
         <property name="text" >
          <string>&amp;Find Next</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="buttonReplaceAll" >
         <property name="text" >
          <string>Replace &amp;All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="buttonClose" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>comboFind</tabstop>
  <tabstop>comboReplace</tabstop>
  <tabstop>checkWords</tabstop>
  <tabstop>checkCase</tabstop>
  <tabstop>checkBackward</tabstop>
  <tabstop>buttonReplace</tabstop>
  <tabstop>buttonReplaceAll</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
